ceagle/api/v1/status.py

```diff
@@ -29,9 +32,17 @@ def get_status(period):
 
 
 @bp_status.route("/health/<period>")
-@fake_status.get_status_health
 def get_status_health(period):
-    return flask.jsonify("fixme!")
+    health_client = client.get_client("health")
+    if not health_client:
+        return flask.make_response(
+            json.dumps({"error": "No health endpoint configured"}),
+            500,
+        )
+
+    api_endpoint = "/api/v1/health"
```

Comment:
this is API endpoint of health api and it should be hardcoded here, since there is no way to devise it from just uri, where api runs